.github/workflows/plugin-check.yml

```diff
@@ -0,0 +1,53 @@
+name: 'WordPress Plugin Checks'
+on: # rebuild any PRs and main branch changes
+  push:
+    branches:
+    - trunk
+    - develop
+
+  pull_request:
+    branches:
+      - trunk
+      - develop
+      - add/plugin-check
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Run plugin check
+      uses: wordpress/plugin-check-action@v1
```

Comment:
I know the PT for this mentions both running this through QIT and running using this Action. Is there any benefit to using QIT instead of doing this directly?

Reply:
The [documentation](https://qit.woo.com/docs/managed-tests/plugin-check/) does not provide guidance on using it as QIT; therefore, we have adopted this approach.